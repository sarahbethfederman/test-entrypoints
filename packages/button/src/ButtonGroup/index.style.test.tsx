import * as React from 'react';
import { mount } from 'enzyme';
import { Wrapper } from './index.style';
import { ButtonSize } from '../Button/index.style';

const sizes: ButtonSize[] = ['lg', 'md', 'sm', 'xs'];

describe('Wrapper', () => {
  sizes.forEach((size) => {
    it(`should have spacing at size "${size}"`, () => {
      const element = mount(<Wrapper size="lg" />);
      expect(element).toMatchSnapshot();
    });
  });
});
