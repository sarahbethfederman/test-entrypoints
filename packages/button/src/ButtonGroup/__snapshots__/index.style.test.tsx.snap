// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper should have spacing at size "lg" 1`] = `
.c0 .c1 + .c1 {
  margin-left: 24px;
}

<styled.div
  size="lg"
>
  <div
    className="c0"
    size="lg"
  />
</styled.div>
`;

exports[`Wrapper should have spacing at size "md" 1`] = `
.c0 .c1 + .c1 {
  margin-left: 24px;
}

<styled.div
  size="lg"
>
  <div
    className="c0"
    size="lg"
  />
</styled.div>
`;

exports[`Wrapper should have spacing at size "sm" 1`] = `
.c0 .c1 + .c1 {
  margin-left: 24px;
}

<styled.div
  size="lg"
>
  <div
    className="c0"
    size="lg"
  />
</styled.div>
`;

exports[`Wrapper should have spacing at size "xs" 1`] = `
.c0 .c1 + .c1 {
  margin-left: 24px;
}

<styled.div
  size="lg"
>
  <div
    className="c0"
    size="lg"
  />
</styled.div>
`;
